components/ui/touchableopacity/SingleClickTouchableOpacity.tsx

```diff
@@ -10,12 +10,13 @@ export const SingleClickTouchableOpacity: React.FC<SingleClickTouchableOpacityPr
   delay = 1000,
   onPress,
   children,
+  disabled = false,
   ...props
 }) => {
   const [processing, setProcessing] = useState(false);
 
   const handlePress = (event: GestureResponderEvent) => {
-    if (onPress && !processing) {
+    if (onPress && !processing && !disabled) {
```

Comment:
this line and 13 are not needed because function handlePress is passed as property to TouchableOpacity which takes care of not firing it when is disabled.

Reply:
it is needed if we remove we can change that value